src/components/form-intro.js

```diff
@@ -0,0 +1,83 @@
+import {nanoid} from 'nanoid';
+import {useState} from 'react';
+import {useForm} from 'react-hook-form';
+
+console.clear();
+
+export default function IntroForm() {
+	const {
+		register,
+		handleSubmit,
+		formState: {errors},
+	} = useForm();
+
+	const [people, setPeople] = useState([
+		{
+			restaurantName: 'Restaurant Name',
+			typeOfFood: 'Type of Food',
+			priceLevel: 'Price Level',
+			id: nanoid(),
+		},
+	]);
+
+	function onSubmit(data) {
+		console.log(data);
+
+		setPeople([...people, {...data, id: nanoid()}]);
+	}
+
+	return (
+		<>
+			<form onSubmit={handleSubmit(onSubmit)}>
+				<input
+					type="text"
+					placeholder="Name of Restaurant"
+					id="restaurantName"
+					{...register('restaurantName', {
+						required: true,
+					})}
+				/>
+
+				{errors.restaurantName && <p>Name of restaurant is required</p>}
+
+				<select id="typeOfFood" {...register('typeOfFood', {required: true})}>
+					<option value="French Cuisine">French Cuisine</option>
+					<option value="Chinese Cuisine">Chinese Cuisine</option>
```

Comment:
Hi Kolja, du konntest die veschiedene values outsourcen und hier drin bloß drüber mappen. 

Würde übersichtlicher aussehen .

Reply:
Hello Petros, thank you for your feedback. I still have the outsourcing on my to-do list for the next user story.